apps/src/codebridge/FileBrowser/hooks/usePrompts.ts

```diff
@@ -0,0 +1,23 @@
+import {useCodebridgeContext} from '@codebridge/codebridgeContext';
+import {openNewFolderPrompt as globalOpenNewFolderPrompt} from '@codebridge/FileBrowser/prompts';
+import {useMemo} from 'react';
+
+import {usePartialApply} from '@cdo/apps/lab2/hooks';
+import {useDialogControl} from '@cdo/apps/lab2/views/dialogs';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+export const usePrompts = () => {
+  const appName = useAppSelector(state => state.lab.levelProperties?.appName);
+  const dialogControl = useDialogControl();
+
+  const {project, newFolder} = useCodebridgeContext();
+
+  const openNewFolderPrompt = usePartialApply(globalOpenNewFolderPrompt, {
+    appName,
+    dialogControl,
+    newFolder,
+    projectFolders: project.folders,
+  });
+
+  return useMemo(() => ({openNewFolderPrompt}), [openNewFolderPrompt]);
+};
```

Comment:
This wrapper hook is going to grow, but is fairly straightforward - it's just going to build the partially applied `*Prompt` functions used in the FileBrowser, so they can all be imported in a hook in one shot.